apps/right-to-rent-check/behaviours/tenants.js

```diff
@@ -144,6 +145,10 @@ module.exports = fields => {
     locals(req, res) {
       const locals = super.locals(req, res);
       locals.tenants = req.sessionModel.get('tenants');
+      _.map(locals.tenants, (tenant) => {
```

Comment:
because you're are not using the returned value from `_.map`, `Array.prototype.forEach` would be more appropriate.